File: Desenvolvimento/server/src/ApplicationCore/Enums/TramitarEmEnum.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Text;

namespace ApplicationCore.Enums
{
    public enum TramitarEmEnum
    {
        [Description("Setores da empresa do processo")]
        SetoresDaEmpresaDoProcesso = 1,

        [Description("Entre setores de todas as empresas")]
        EntreSetoresDeTodasAsEmpresa = 2,

        [Description("Fluxo definido")]
        FluxoDefinido = 3
    }
}

File: Desenvolvimento/client/src/@fuse/types/models/enums/tramitar-em-enum.ts
export class TramitarEm {
    constructor(
        public key: any,
        public value: string
    ) { }
}

export enum TramitarEmEnum {
    SetoresDaEmpresaDoProcesso = 1,
    EntreSetoresDeTodasAsEmpresa = 2,
    FluxoDefinido = 3
}

export const TramitarEmEnumArrayKeyValue = [
    new TramitarEm('', 'Todos'),
    new TramitarEm(TramitarEmEnum.SetoresDaEmpresaDoProcesso, 'Setores da empresa do processo'),
    new TramitarEm(TramitarEmEnum.EntreSetoresDeTodasAsEmpresa, 'Entre setores de todas as empresas'),
    new TramitarEm(TramitarEmEnum.FluxoDefinido, 'Fluxo definido'),
];
